### PR TITLE
created test_item.py and modified files accordingly

### PR DESCRIPTION
**Pull request by anuva98**
Changes:
1. Added test_item.py
2. deleted test_models.py
3. added a few lines in deserialize block for item.py

### CODE REVIEW

**Review by rheachandok — APPROVED** [review]
lgtm

File: service/models/item.py [hunk 1]
```diff
@@ -33,9 +33,12 @@ def serialize(self):
     def deserialize(self, data):
         """Populates an Item from a dictionary"""
         try:
+            self.order_id = data["order_id"]
             self.product_name = data["product_name"]
             self.quantity = data["quantity"]
             self.price = data["price"]
+            self.created_at = data["created_at"]
```
Comment (Hyxpillow) [inline service/models/item.py:40]: Oh, I realized that the "created_at" and "updated_at" attributes should also not be checked during deserialization, for the same reason as yesterday's issue. 
So it's not just here, the deserialization of order might need to be corrected as well.

**Review by Hyxpillow — CHANGES_REQUESTED** [review]

File: service/models/item.py (hunk 1 above)
Comment (Hyxpillow) [reply service/models/item.py:40]: These two attributes should be automatically set by the database, not user. So the user's json should not contain them.

Comment (Hyxpillow) [reply service/models/item.py:40]: Sorry for not realizing it yesterday.

Comment (anuva98) [reply service/models/item.py:40]: Oh god!! That makes so much sense. I'm sorry I've been a bit busy so I didn't understand exactly why that was missing. Thank you for pointing it out. I'll change the order one to as part of this PR.

**Review by Hyxpillow — APPROVED** [review]

**Pull request merged**